comms/dht/migrations/2021-07-21-0846252_add_dedup_cache/up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE dedup_cache (
```

Comment:
Should we not add the sender address in here for inspection purposes?

Reply:
Good idea